cmd/snap/main.go

```diff
@@ -92,6 +94,17 @@ type parserSetter interface {
 // Since commands have local state a fresh parser is required to isolate tests
 // from each other.
 func Parser() *flags.Parser {
+	optionsData.Version = func() error {
+		cv, err := Client().Version()
+		if err != nil {
+			cv = i18n.G("unavailable")
```

Comment:
it's not a client error if there isn't a snapd installed/running/accessible, and we should still report the client version in that case